bool front
bool front_left
bool front_right
bool rear_left
bool rear_right
bool back

float32 left_theta
float32 right_theta
float32 left_hypo_length
float32 right_hypo_length

float32 front_distance
float32 front_left_distance
float32 rear_left_distance
float32 front_right_distance
float32 rear_right_distance
float32 back_distance
